ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -17,23 +22,37 @@
  * Created by nata on 24.06.15.
  */
 public class AddFormTest extends TestBase {
+    private static String name = "newFormTest1";
+    private static String description = "description of new form";
+    private static  String version = "1.2";
     private HomePage homePage;
     private HeaderPage headerPage;
     private ManageFormsPage manageForm;
+    private AdministrationPage administrationPage;
     private ClinicianFacingPatientDashboardPage patientDashboardPage;
+    private ManageHtmlFormsPage manageHtmlFormsPage;
+    private HtmlFormsPage htmlFormsPage;
 
     @Before
     public void setUp() throws Exception {
         homePage = new HomePage(page);
-        assertPage(homePage);
+        administrationPage =  new AdministrationPage(page);
         headerPage = new HeaderPage(driver);
+        htmlFormsPage = new HtmlFormsPage(page);
+        manageHtmlFormsPage = new ManageHtmlFormsPage(page);
         manageForm = new ManageFormsPage(driver);
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
     }
 
-    @Ignore //ignore due to moving forms functionality
     @Test
+    @Category(BuildTests.class) 
     public void addFormTest() throws Exception {
+      manageHtmlFormsPage =  homePage.goToAdministration().clickOnManageHtmlForms();
+      if (manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '"+ name + "')]")).isEmpty()){
+    	   manageHtmlFormsPage.clickOnNewHtmlForm();
+          htmlFormsPage.createNewFormTest(name,description,version);
+      }
+    	homePage.go();
```

Comment:
please fix the code miss-formatting here

Reply:
Resolved thanks